loops/Loops Practice/css/style.css

```diff
@@ -41,4 +41,4 @@ h2 {
 p {
     font-size: 2em;
     line-height: 0.8;
-}
\ No newline at end of file
+}
```

Comment:
For the directory we don't want to have spaces in it. So it should be
`loops/loopsPractice` or `loops/loops-practice` - I've seen examples of both.

Reply:
Fixed in latest commit